x-pack/plugins/ml/public/services/ml_api_service/filters.js

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+// Service for querying filters, which hold lists of entities,
+// for example a list of known safe URL domains.
+
+import chrome from 'ui/chrome';
+
+import { http } from 'plugins/ml/services/http_service';
+
+const basePath = chrome.addBasePath('/api/ml');
+
+export const filters = {
+  getFilters() {
```

Comment:
the convention we've followed in the past is to have one function named e.g. `filters` which takes an optional id list.